runtime/lua/vim/lsp.lua

```diff
@@ -71,6 +71,12 @@ lsp._request_name_to_capability = {
   [ms.textDocument_didOpen] = { 'textDocumentSync', 'openClose' },
   [ms.textDocument_willSave] = { 'textDocumentSync', 'willSave' },
   [ms.textDocument_willSaveWaitUntil] = { 'textDocumentSync', 'willSaveWaitUntil' },
+  [ms.textDocument_documentColor] = { 'colorProvider' },
```

Comment:
Super nit which should be a separate commit, but to better keep track of what's been added so far it would be useful to sort these keys alphabetically.

Reply:
Definitely! I will.

Reply:
Done. Thanks!

https://github.com/neovim/neovim/pull/30098/commits/06246472b3da7351a06206e19525f8081ba239b4